src/privileged/notificationBar/api.js

```diff
@@ -49,38 +49,47 @@ class NotificationBarEventEmitter extends EventEmitter {
   emitShow(variationName) {
     const self = this;
     const recentWindow = getMostRecentBrowserWindow();
-    const doc = recentWindow.document;
-
     const browser = recentWindow.gBrowser.selectedBrowser;
     const tabId = tabTracker.getBrowserTabId(browser);
 
     const primaryAction =  {
-      label: "Yes!",
+      disableHighlight: true,
+      label: "Yes (broken)",
       accessKey: "f",
       callback: () => {
-        self.emit("page-broken", {tabId});
+        self.emit("page-broken", tabId);
       },
     };
 
     const secondaryActions =  [
       {
-        label: "Nope",
+        label: "No (works)",
         accessKey: "d",
         callback: () => {
-          self.emit("page-not-broken", {tabId});
+          self.emit("page-not-broken", tabId);
         },
       },
     ];
 
-    const populatePanel = (event) => {
-      // console.log("EVENT HAPPENED", event);
-      // Can edit the contents of the panel here if we want variations per branch
-      // if (event !== "shown") {
-      //   return;
-      // }
-    };
+    // option persistant is described as: "the notification will always
+    //                    persist even across tab and app changes (but not across
+    //                    location changes), until the user accepts or rejects
+    //                    the request. The notification will never be implicitly
+    //                    dismissed."
+    // This is not the case, it does not persist across tab changes
```

Comment:
I think what's meant is that the notification will reappear after leaving the tab and coming back, I would happily take a patch to m-c that improves this documentation :)